presto-main/src/main/java/com/facebook/presto/memory/ClusterMemoryManager.java

```diff
@@ -260,16 +260,23 @@ public synchronized void process(Iterable<QueryExecution> runningQueries, Suppli
         clusterUserMemoryReservation.set(totalUserMemoryBytes);
         clusterTotalMemoryReservation.set(totalMemoryBytes);
 
-        if (!(lowMemoryKiller instanceof NoneLowMemoryKiller) &&
+        boolean killOnOOMDelayPassed = nanosSince(lastTimeNotOutOfMemory).compareTo(killOnOutOfMemoryDelay) > 0;
```

Comment:
There are variables using both `OOM` and `Oom`. Both are reasonable to me, just choose either one and be consistent.